src/data/markdown/docs/02 javascript api/07 k6-experimental/04 tracing/01 instrumentHTTP.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'instrumentHTTP'
+excerpt: 'instrumentHTTP instruments the k6 http module with tracing capabilities.'
+---
+
+The `instrumentHTTP` function instruments the k6 http module with tracing capabilities. It will transparently replace each of the k6 http module [del](/javascript-api/k6-http/del), [get](/javascript-api/k6-http/get), [head](/javascript-api/k6-http/head), [options](/javascript-api/k6-http/options), [patch](/javascript-api/k6-http/patch), [post](/javascript-api/k6-http/post), [put](/javascript-api/k6-http/head), and [request](/javascript-api/k6-http/request) functions with versions that will automatically attach a trace context to every request.
+
+From a user perspective, the `instrumentHTTP` allows automatically adding tracing information to HTTP requests without having to rewrite any existing code. Calling it once in the init context will ensure that all requests made by the HTTP module from that point forward will have a [trace context header](https://www.w3.org/TR/trace-context/) added to them and will add the used `trace_id` as part of the produced data points output metadata.
```

Comment:
> Calling it once in the init context will ensure that all requests made by the HTTP module from that point forward will have a **trace context header**...

I don't think this is true. Jaeger defaults to its own [specification](https://www.jaegertracing.io/docs/1.21/client-libraries/#propagation-format).

Reply:
You're right 🤦🏻 I linked to the W3C specification automatically, but in this case it doesn't work, gonna change it indeed 🙇🏻